### PR TITLE
3676 - drying up code

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js [hunk 1]
```diff
@@ -164,40 +168,14 @@ exports.setNoticesForCalendaredTrialSessionInteractor = async ({
...
-        );
-        copiedPages.forEach(page => {
-          newPdfDoc.addPage(page);
-        });
-      }
+
+      await appendPaperServiceAddressPageToPdf({
+        applicationContext,
+        caseEntity,
+        newPdfDoc,
```
Comment (kkoskelin) [inline shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js:175]: where is newPdfDoc getting set here?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
One thing I'm not sure about -- I think maybe needs a different name/identifier.

File: shared/src/business/utilities/appendPaperServiceAddressPageToPdf.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+const { PDFDocument } = require('pdf-lib');
+
+exports.appendPaperServiceAddressPageToPdf = async ({
```
Comment (rachaelparris) [inline shared/src/business/utilities/appendPaperServiceAddressPageToPdf.js:3]: I think this and the other util you added should probably be part of `useCaseHelpers` so they can be mocked out for testing, and we can just test this piece by itself. I won't hold up this PR for that, but I think it probably should be on our radar for refactoring for this story.

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js (hunk 1 above)
Comment (codyseibert) [reply shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js:175]: that comes from somewhere above, outside the function

Comment (kkoskelin) [reply shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js:175]: my bad! wasn't reading indents properly.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**